### ISSUE TITLE
Come up with ARM/bicep templates

### ISSUE THREAD

**Issue by stijnmoreels** [arcus-azure/arcus#215]
**Is your feature request related to a problem? Please describe.**
Some libraries/functionality require additional Azure setup before it can be used.

**Describe the solution you'd like**
It would be great if we can create ARM/bicep templates for people to start off quick and setting as much of the necessary setup as possible.
We should first run over the places where this could be used.

**Additional context**
First re-discovered in https://github.com/arcus-azure/arcus.eventgrid/pull/232#discussion_r860719188

**Comment by fgheysels** [comment]
I guess this is ongoing ?

**Comment by stijnmoreels** [comment]
> I guess this is ongoing ?

Yeah, security already has been moved, but others are still in process.

- [x] Arcus security
- [x] Arcus testing
- [ ] Arcus messaging
- [ ] Arcus observbility
- [ ] Arcus scripting
- [ ] Arcus templates
- [ ] Arcus web API
- [ ] ~Arcus event grid~
- [ ] ~Arcus ML~
- [ ] Arcus Azure ML~
